### DISCUSSION TITLE
Data synchronization problem, or how to back up data?

### DISCUSSION THREAD

**Discussion by zhoumao1** [AutoDarkMode/Windows-Auto-Night-Mode#839] [Q&A] [answered]
What if I change computers with the same account under win11, how do I keep my settings the same? In other words, how do I back up my data?

**Comment by Armin2208** [comment, marked as answer]
Richt click the Tray Icon of Auto Dark Mode → Open Config Directory → copy the files to the new computer. 

The most important files are config.yaml and maybe scripts.yaml. But you can just transfer all of them.

**Reply by zhoumao1** [reply]
Thank you very much for your reply, I know exactly how to back up now!
